pom.xml

```diff
@@ -398,6 +398,11 @@
         <scope>test</scope>
         <version>1.7.29</version>
       </dependency>
+      <dependency>
+        <groupId>log4j</groupId>
+        <artifactId>log4j</artifactId>
+        <version>1.2.17</version>
+      </dependency>
```

Comment:
Make sense, but when we run the command, always output
```
[dcadmin@dcadmin-work apache-ratis-3.0.0-SNAPSHOT-bin]$ bin/ratis sh
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Usage: ratis sh [generic options]
	 [election [transfer] [stepDown] [pause] [resume]]         
	 [group [info] [list]]                                     
	 [peer [add] [remove] [setPriority]]                       
	 [snapshot [create]]       
```

Reply:
@leo65535 , Indeed, ratis-shell should not use log4j at all since it won't print any log message.  We should disable it.

We should have `slf4j-nop.jar` as described in https://www.slf4j.org/codes.html#StaticLoggerBinder

Reply:
Or we may try `slf4j-simple.jar`.

Reply:
Got, done.